docs/reference.rst

```diff
@@ -50,119 +72,9 @@ Camera objects
     :member-order: bysource
```

Comment:
I guess you will move the Camera objects to pygfx.cameras subpage?

Reply:
@Korijn  Yes that is the plan. I didn't touch them yet because we said they are due for a refactor, but I will add the plumbing  to keep the current classes documented.

Reply:
I think that was about Controllers, not Cameras, actually. :)

Reply:
Both, to some extend, probably ...